test/ReactWrapper-spec.jsx

```diff
@@ -929,8 +929,9 @@ describeWithDOM('mount', () => {
       }
 
       const similarException = ((() => {
+        const user = {};
         try {
-          return undefined.givenName;
+          return user.name.givenName;
```

Comment:
Firefox's error message for the error in the component in this test is different from chrome's/node's, if I remember correctly it's something like "cannot read property givenName of user.name", while in chrome is "cannot read property givenName of undefined". that's why I made this change, to ensure the property/variable names are the same in `similarException` and in the component